app/controllers/katello/api/v2/content_view_versions_controller.rb

```diff
@@ -60,6 +60,13 @@ def promote
       respond_for_async :resource => task
     end
 
+    api :PUT, "/content_view_versions/:id/republish_repositories", N_("Forces a republish of the version's repositories' metadata.")
+    param :id, :identifier, :desc => N_("Content view version identifier"), :required => true
+    def republish_repositories
+      task = async_task(::Actions::BulkAction, ::Actions::Katello::Repository::MetadataGenerate, @version.repositories, :force => true)
```

Comment:
Agreed on history, but seems like it should be added to the list of tasks (on Tasks tab); otherwise, user does have to go filtering on Monitor->Tasks.